playground/DEPLOY.md

```diff
@@ -0,0 +1,95 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+## Manual App Install
+To install Playground applications into existed GCP account you'll need to follow several steps, in this document just simple set of commands will be provided that will guide you over install process.
+
+Before starting that process, you will need to create GCP Artifact Registry instance to store there docker files that would be deployed. Guide to install that registry could be found in [that](https://github.com/apache/beam/blob/master/playground/terraform/README.md) document.
```

Comment:
```suggestion
### Getting Started

You will need to create an GCP Artifact Registry instance to host Docker image files used to deploy the Playground services.
```

It's unclear to me if the recommendation here is to create them using Terraform by following https://github.com/apache/beam/blob/master/playground/terraform/README.md#creating-gcp-resources-using-terraform - if so, can we specifically link here and say 

"""
Create the repository in Terraform by following this [guide](https://github.com/apache/beam/blob/master/playground/terraform/README.md#creating-gcp-resources-using-terraform. 
"""